tests/dialects/test_postgres.py

```diff
@@ -99,6 +105,12 @@ def test_ddl(self):
                 "postgres": "CREATE TABLE products (product_no INT, name TEXT, price DECIMAL, UNIQUE (product_no, name))"
             },
         )
+        self.validate_all(
+            "WITH t(c) AS (SELECT 1) SELECT * INTO TEMP UNLOGGED foo FROM (SELECT c AS c FROM t) AS temp",
+            write={
+                "postgres": "WITH t(c) AS (SELECT 1) SELECT * INTO TEMPORARY foo FROM (SELECT c AS c FROM t) AS temp",
+            },
+        )
```

Comment:
This test isn't quite correct, you can't have both `TEMP` and `UNLOGGED` in the same statement.

```suggestion
```